lib/post/widgets/post_view.dart

```diff
@@ -173,16 +184,32 @@ class _PostSubviewState extends State<PostSubview> with SingleTickerProviderStat
                 ),
                 expanded: Padding(
                   padding: const EdgeInsets.only(top: 8.0),
-                  child: widget.viewSource
-                      ? ScalableText(
-                          post.body ?? '',
-                          style: theme.textTheme.bodySmall?.copyWith(fontFamily: 'monospace'),
-                          fontScale: thunderState.contentFontSizeScale,
-                        )
-                      : CommonMarkdownBody(
-                          body: post.body ?? '',
-                          isSelectableText: widget.selectable,
-                        ),
+                  child: ConditionalParentWidget(
```

Comment:
Do you think we need to apply the `SelectableRegion` here, or would it be more suited as part of the create comment page? (same thing with the comment SelectableRegion)

I guess the main discussion point is whether or not we're expecting to make the post selectable in other areas outside of the create comment page. We already have a way to copy post/comments from #1327!

Reply:
I originally put the `SelectableRegion` around the whole `PostSubview` and `CommentContent` widgets, but that had a couple of weird side-effects. For one, I couldn't see the drag handles or the selection highlight color. But the bigger issue is that it allows everything in those widgets to be selectable, including the metadata, which I thought was weird! It seemed to be behave much better when wrapped directly around the `CommonMarkdownBody`, and I don't think it causes any harm, even if this is the only place we use it. Thoughts?

Reply:
Ahh okay, you make a good point about allowing metadata to be selected! We'll keep it as it!